src/overlays/actors/ovl_En_Raf/z_en_raf.c

```diff
@@ -334,6 +332,7 @@ void EnRaf_Idle(EnRaf* this, GlobalContext* globalCtx) {
 
         explosive = globalCtx->actorCtx.actorLists[ACTORCAT_EXPLOSIVES].first;
         while (explosive != NULL) {
+            // This check is pointless, since EnRaf never spawns as an explosive.
```

Comment:
I think I'd just say
```suggestion
            // This check is pointless, since EnRaf is never in the explosive category.
```